aptos-move/framework/aptos-framework/sources/account.spec.move

```diff
@@ -1,6 +1,145 @@
 spec aptos_framework::account {
+    
+    spec module {
+        pragma verify = true;
+        pragma aborts_if_is_strict;
+    }
+
     spec create_signer {
-        // TODO: temporary mockup.
         pragma opaque;
+        aborts_if [abstract] false;
+        ensures [abstract] signer::address_of(result) == addr;
+    }
+
+    spec initialize {
+        aborts_if !system_addresses::is_aptos_framework_address(signer::address_of(aptos_framework));
+        aborts_if exists<OriginatingAddress>(signer::address_of(aptos_framework));
+        ensures exists<OriginatingAddress>(signer::address_of(aptos_framework));
+    }
+
+    spec create_account {
+        pragma aborts_if_is_partial;
+        aborts_if exists<Account>(new_address);
+        aborts_if new_address == @vm_reserved || new_address == @aptos_framework || new_address == @aptos_token;
```

Comment:
This is an incomplete spec, because it says nothing about the state of the memory after we created the account. Please add a TODO to complete this.

Reply:
Done